quota/redis/redistb/embed_redis.go

```diff
@@ -0,0 +1,75 @@
+// +build ignore
+
+// This is a helper utility to embed the Redis Lua scripts into a Go source
+// file.
+package main
+
+import (
+	"bytes"
+	"go/format"
+	"io/ioutil"
+	"log"
+	"os"
+	"strconv"
+	"text/template"
+)
+
+var packageTemplate = template.Must(template.New("").Parse(`
+// Code generated by quota/redis/redistb/gen.go. DO NOT EDIT.
+// source: {{ .Filename }}
+
+package redistb
+
+import (
+	"github.com/go-redis/redis"
+)
+
+// contents of the '{{ .Prefix }}' Redis Lua script
+const {{ .Prefix }}ScriptContents = {{ .Content }}
+
+// Redis Script type for the '{{ .Prefix }}' Redis lua script
+var {{ .Prefix }}Script = redis.NewScript({{ .Prefix }}ScriptContents)
+`))
+
+type templateData struct {
+	Prefix   string
+	Filename string
+	Content  string
+}
+
+func main() {
+	if len(os.Args) != 4 {
+		log.Fatalf("usage: %s prefix file.lua output.go", os.Args[0])
```

Comment:
`log.Exitf` will skip the stacktrace - likely fine to use throughout the `main` func.

Reply:
Ah, this is a [`glog`](https://github.com/golang/glog) thing :(